scripts/check-golint.sh

```diff
@@ -18,6 +18,14 @@ set -e
 set -u
 set -o pipefail
 
+# Set a GOBIN to install golint.
+ROOT_DIR="$(cd "$(dirname "${BASH_SOURCE}")/.." && pwd -P)"
+export GOBIN="${ROOT_DIR}/_output/local/bin"
```

Comment:
you may want to git ignore the `_output` directory, too.

Reply:
@nikhiljindal definitely want to make sure you're doing this, I don't see any .gitignore currently

Reply:
I kind of think this is the first PR that exhibits degraded coverage data, #190. Perhaps due to changing the path? Does this PR change where/what "go coverage" is executed?

Reply:
Sent https://github.com/GoogleCloudPlatform/k8s-multicluster-ingress/pull/191 to add .gitignore as per comments here.